src/devices/cpu/powerpc/ppcdrc.cpp

```diff
@@ -3926,20 +3926,20 @@ void ppc_device::log_opcode_desc(drcuml_state *drcuml, const opcode_desc *descli
 	/* output each descriptor */
 	for ( ; desclist != nullptr; desclist = desclist->next())
 	{
-		char buffer[100];
+		std::string buffer;
```

Comment:
Could you turn this into some kind of stream object and lift it out of the loop?  Then allocations can be saved.